### PR TITLE
First draft of manuscripts section

### PR DESCRIPTION
**Pull request by lexnederbragt**

### CODE REVIEW

**Comment by kcranston** [comment]
Nicely done! There isn't an introduction to this section, and I wonder if it needs one. Thoughts that might go in there:
- ability to use the same tools and workflows for manuscript preparation as for the research itself (writing is version controlled, collaborative, reproducible, etc). 
- a recommendation about having all authors agree on a writing workflow in advance of starting the manuscript (or perhaps this belong in the collaboration section?)

File: index.md [hunk 1]
```diff
@@ -301,78 +301,54 @@ should therefore not be put in a version control repository.
 
 ## Manuscripts
 
+Many of the the same tools and workflows we describe for research will be equally beneficial for manuscript preparation: making the writing version controlled, collaborative, and reproducible. It is important, however, to have all authors agree on a particular collaborative writing workflow in advance of starting the actual writing of the manuscript. Once you agree, as lead author, don't hesitate to lay out some ground rules that you expect the others to adhere to - even if they are many years your senior.
+
 ### Goals
 
-1.  Make text accessible to yourself and others now and in the future
+A common, but unfortunate, practice is that the lead author at different points in the writing process sends around versions to coauthors for collecting feedback, and receiving multiple documents back with comments. This leads to a lot of files to keep track of and much manual labor to merge all the comments into the updated 'master' document. This section describes alternatives to this workflow that will satisfy the following goals:
```
Comment (gvwilson) [inline index.md]: I would move this up above the Goals heading as motivation: here's what we're trying to _avoid_.

File: index.md [hunk 2]
```diff
@@ -301,78 +301,54 @@ should therefore not be put in a version control repository.
...
+
+1.  Make text accessible to yourself and others now and in the future by using a single entry point for the master document that is accessible to all coauthors at all times.
```
Comment (gvwilson) [inline index.md]: Our readers won't know what "entry point" means in this context.

File: index.md [hunk 3]
```diff
@@ -301,78 +301,54 @@ should therefore not be put in a version control repository.
...
-    2.  Tools needed to compile manuscripts are managed just like tools used to do simulation or analysis
-*   *Or:*
-    1.  Manuscripts are written using Google Docs or some other online tools with rich formatting and change tracking
-    2.  A short text file is added to the `doc` directory with metadata about each online manuscript
-        *   Just as the `data` directory might contain links rather than actual data
-    3.  The manuscript is downloaded and saved in `doc` in an editable form (e.g., `.docx` or `.odt`) after major changes
-*   Either way, use a distributed web-based system for managing the manuscript
-    so that the master document is clearly defined and everyone can collaborate on an equal footing
+1.  a manuscript is written using Google Docs or some other online tool with rich formatting, change tracking, and reference management.
+2. In the project folder, add a short text file to the `doc` directory with metadata about each online manuscript (e.g. the URL). This is analogous to the `data` directory, which might contain links to the location of the datafile(s), rather than the actual files.
```
Comment (gvwilson) [inline index.md]: What should this file be called?  We're making specific recommendations elsewhere about filenames.

File: index.md [hunk 4]
```diff
@@ -301,78 +301,54 @@ should therefore not be put in a version control repository.
...
 
-> BK: I'm still not convinced about the necessity to track the manuscript for a single author.
->
-> GW: The reviews I'm getting on this outline, and the ease of doing pre-commit review, convinces me.
->
-> BK: That's because you have friends who care about and understand what you are writing.
-> This intersection is often empty for, say, a grad student.
->
-> GW: Maybe if we provide better tools, we can help fix that.
+As an example of an implementation of these recommendations, this paper was written using a central online repository (Github), the *issue* functionality for discussing the outline and text, and *pull requests* for reviewing the contributions from different authors and other contributors before merging them in.
```
Comment (gvwilson) [inline index.md]: We should say somewhere that there should be one way to provide feedback (as well as just one way to author), i.e., settle on issues, a mail thread, or _something_ as the One True Channel.

**Comment by gvwilson** [comment]
ETA for merge?

**Comment by lexnederbragt** [comment]
Tomorrow, most likely.

**Pull request merged**